src/store/annotations/selectors.ts

```diff
@@ -4,6 +4,7 @@ import { AppState } from '../types';
 
 type State = Pick<AppState, 'annotations'>;
 
+export const areAnnotationsInitialized = ({ annotations }: State): boolean => annotations.isInitialized;
```

Comment:
`getIsInitialized`?

Reply:
Sure, that works too. Since returning a boolean wanted to leave it as a question to the state object. But that works too...